### DISCUSSION TITLE
Does rspack have a `rspack-chain`?

### DISCUSSION THREAD

**Discussion by yzqdev** [web-infra-dev/rspack#6452] [Q&A] [answered]
https://rsbuild.dev/guide/basic/configure-rspack#toolsbundlerchain

rsbuild has a toolsbundlerchain to config built in rspack. so does rspack have a bundler chain builtin like webpack-chain?

maybe like?
```ts
const rspack=require("@rspack/core")
const chain= rspack.ConfigObj()
chain.module
        .rule('md')
        .test(/\.md$/)
        .use('md-loader')
        .loader('md-loader');
 
module.exports = config.toConfig();
```

**Comment by xc2** [comment, marked as answer]
webpack-chain works fine for rspack, please try out it first.

**Reply by watsonhaw5566** [reply]
你好，请教一下,您回复中的 webpack-chain 指的是 [webpack-5-chian](https://www.npmjs.com/package/webpack-5-chain) 还是 [webpack-chain](https://github.com/neutrinojs/webpack-chain) 的版本？谢谢。

**Reply by xc2** [reply]
webpack-5-chain, which is also used by rsbuild.

**Reply by yzqdev** [reply]
Thank you, i'll try.

**Reply by watsonhaw5566** [reply]
Tanks my bro
